tartiflette/schema/schema.py

```diff
@@ -1058,3 +1060,31 @@ def bake_execute(self, func_query, func_subscription):
         for type_name, type_definition in self.type_definitions.items():
             if not type_name.startswith("__"):
                 self.types.append(type_definition)
+
+    async def bake(
+        self,
+        custom_default_resolver: Optional[Callable] = None,
+        custom_default_type_resolver: Optional[Callable] = None,
+    ) -> None:
+        """
+        Bake the final schema (it should not change after this) used for
```

Comment:
@abusi  I'm wondering if it MUST not change after this stage?  (the schema)

Reply:
Yep, because if the schema changes, it needs to be re-baked. But don't worry, this don't work yet It's really really early work.